Documentation/bpf/toolchain.rst

```diff
@@ -1864,7 +1864,7 @@ of all details, but enough for getting started.
     # tc qdisc del dev em1 clsact
 
   tc BPF programs can also be offloaded if the NIC and driver has support for it
-  similarly as with XDP BPF programs. Netronome's nfp supported NICs offer both
+  similarly, as with XDP BPF programs. Netronome's nfp supported NICs offer both
```

Comment:
Doesn't sound correct. I think this should be “... if the NIC and driver have support for it, like XDP BPF programs.”